src/components/Dashboard.vue

```diff
@@ -33,6 +35,24 @@
         @updateRequirementsMenu="updateRequirementsMenu"
       />
     </div>
+    <tourwindow
+      :title="welcome"
+      :text="welcomeBodytext"
+      :exit="welcomeExit"
+      :buttonText="welcomeButtonText"
+      v-on:hide = "welcomeHidden = false; if(startTour==false) startTour = true"
```

Comment:
Couple things
1. There is a shorthand for 'v-on' - you can just use '@' (see https://vuejs.org/v2/guide/syntax.html#v-on-Shorthand)
2. Might be a bit nicer to move everything in this v-on to a function to be a bit more readable and easier to change later.